pybricks/tools/pb_module_tools.c

```diff
@@ -143,10 +144,88 @@ STATIC mp_obj_t pb_module_tools_run_task(size_t n_args, const mp_obj_t *pos_args
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_KW(pb_module_tools_run_task_obj, 1, pb_module_tools_run_task);
 
+#if PYBRICKS_PY_COMMON && PYBRICKS_PY_COMMON_LIGHT_MATRIX
```

Comment:
```suggestion
#if PYBRICKS_PY_TOOLS_HUB_MENU
```

If given infinite time, I would like to use Kconfig to express complex relations of config options. But for now, I think it is best to just keep things simple and make unique config options for each feature use this sort of pattern.

```c
#if THIS_FEATURE
#if !DEPENDENT_FEATURE
#error "THIS_FEATURE requires DEPENDENT_FEATURE"
#endif

...

#endif // THIS_FEATURE
```